snipeit/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: snipeit
-version: 2.0.0
+version: 2.1.0
```

Comment:
Raise to 2.2.0

Reply:
I have raised the version to 2.2.0, but it is conflicting with the current master.